packages/renderer/src/lib/kube/pods/PodColumnActions.spec.ts

```diff
@@ -0,0 +1,40 @@
+/**********************************************************************
```

Comment:
I am wondering if we should not try to use a better folder structures instead of prefixing all component with `PodColumn` let's just create a `lib/kube/pods/column` folder and add `Actions.svelte`, `Containers.svelte` etc.

Reply:
That's actually a pretty good idea. I don't want to start that here if it's not agreed in general, but if we get a +1 or two I'll start that here and open an issue for the others.